R/method-correlation.R

```diff
@@ -1,3 +1,10 @@
+## prefilter predicates
+halfOfSamplesHaveNonZeroAbundance <- function(x){sum(x > 0) >= length(x)/2}
```

Comment:
this predicate is more general than "samples" and "abundances". Can we generalize the name to something like "halfAreNonZero"?